Tools/c-analyzer/cpython/_parser.py

```diff
@@ -89,6 +89,9 @@ def clean_lines(text):
 # not actually source
 Python/bytecodes.c
 
+# Fuzzers, these are meant to have global state.
+Modules/_xxtestfuzz/fuzzer.c
```

Comment:
What, there is a third ignored list?

I think it is better to add it in `Tools/c-analyzer/cpython/ignored.tsv` (where old names are already added).

Reply:
Oh thanks, I missed that they're already in `ignored.tsv`. Renamed the old names there.